.github/workflows/publish-code.yml

```diff
@@ -33,7 +33,7 @@ jobs:
           dotnet-version: '5.0.x'
           source-url: https://api.nuget.org/v3/index.json
         env:
-          NUGET_AUTH_TOKEN: ${{secrets.NUGET_AUTH_TOKEN}}
+          NUGET_AUTH_TOKEN: ${{secrets.RELAY_NUGET_TOKEN}}
```

Comment:
@Shane32 Why other secret here? I would like to have unified secrets across all org repos.

Reply:
I can’t change org secrets, and the token needed to be changed

Reply:
Maybe we will return to this later. While it does not matter. I just deal with tons of mail.